tests/test_locality_NeighborQuery.py

```diff
@@ -205,10 +206,8 @@ def test_query_ball_to_nlist(self):
         L = 10  # Box Dimensions
         N = 400  # number of particles
 
-        box = freud.box.Box.cube(L)
-        np.random.seed(0)
-        ref_points = np.random.rand(N, 3)*L
-        points = np.random.rand(N, 3)*L
+        box, ref_points = make_box_and_random_points(L, N, seed=0)
+        _, points = make_box_and_random_points(L, N, seed=1)
```

Comment:
This is a little awkward, can you just call `np.random.rand(N, 3)*L` for the points arrays in this file instead?